tests/helpers.py

```diff
@@ -31,6 +32,14 @@
 test_account3 = Account.from_key(os.environ.get("EVENTS_TESTS_PRIVATE_KEY3", None))
 
 
+class MetadataStates(IntEnum):
```

Comment:
this should not be in tests, please move it somewhere more visible. We might need it in other places.